src/System.Runtime.InteropServices.WindowsRuntime/src/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcore50aot-Windows_NT;
+      netstandard1.3-Windows_NT;
```

Comment:
We don't need to be.  You can imagine it the same as a nuget package.  The nuget package can have a TFM only asset as well as RID+TFM asset.

As for build-all, yes it will build both, but that isn't a problem.  For the build-all scenario I'm expecting it to build everything, not scoped by OSGroup.